hazelcast/src/main/java/com/hazelcast/map/impl/operation/ContainsValueExceptKeysOperation.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
```

Comment:
Please update all new headers to `Copyright (c) 2008-2018...`. I promise we will merge this PR in this year :), hopefully in the next couple of days.